server.js

```diff
@@ -0,0 +1,33 @@
+// Modules
+const express = require('express');
+const server = express();
+const mongoose = require('mongoose');
+const helmet = require('helmet');
```

Comment:
Thanks for clarifying where this mystery helmet came from! I will have to let Ryan you picked it up! Nice work thinking outside the box.